src/passwords.js

```diff
@@ -15,7 +18,7 @@ module.exports = {
    * @return {Promise<String>} hashed password
    */
   hash: function(password) {
-    return bcrypt.hash(password, 10);
+    return bcrypt.hash(password, rounds);
```

Comment:
According to the documentation, the 'rounds' argument is actually the length of the salt and not the number of rounds.
https://github.com/dcodeIO/bcrypt.js#hashs-salt-callback-progresscallback

genSalt is the function that takes the number of rounds.

Reply:
The documentation disagrees with their implementation. The second argument is fed to genSalt as the number of rounds.

Reply:
I see that looking at the implementation now. I still that we should pass in a rounds of undefined and let the library pick the default rather than us providing it.

Reply:
I agree that leaving it up to the library is a good idea. I can look for test performance gains elsewhere

Reply:
I don't have an issue with lowering the number of rounds for testing purposes. I just didn't like the idea of specifying it explicitly as our default (so in my opinion the unpatched version is flawed).

Reply:
I've updated this PR setting bcryptRounds to `0` which will cause the library to default to 10. Unfortunately `bcrypt.hash(password)` and `bcrypt.hash(password, null)` don't work with bcrypt.js and `bcrypt.hash(password, undefined)` causes `config` to complain